Automattic/jetpack
# Provide Jetpack Version for BlazePress App

Main ticket is in Tumblr's Jira: `SELFSERVE-815`



Recently we introduced config for BlazePress App that provides initial data for the App in https://github.com/Automattic/jetpack/pull/30103.

This PR adds providing the current Jetpack's version to the same config.




## Proposed changes:

* Populate Jetpack's version in `'jetpack_version'` node.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Start Jetpack locally, use Jurassic Tube
- Open Tools > Advertising (if you don't see that menu item, re-establishing Jetpack connection might help)
- Open your browser's Dev Tools / Console
- Run `window.configData.initial_state.sites.items[window.configData.blog_id].options.jetpack_version` and ensure that you see your Jetpack plugin's version like in a sample below:

<img width="650" alt="image" src="https://github.com/Automattic/jetpack/assets/115007291/3f9986b3-3778-4243-b3a0-572f4185ff9d">
